content/spire/getting-started.md

```diff
@@ -169,15 +169,17 @@ To configure the trust bundle on the agent side, edit the configuration file  so
 
 1. Edit the agent’s configuration file in  **/opt/spire/conf/agent/agent.conf**
 2. Locate the **trust_bundle_path = { .. }** entry 
-3. Set the value to  "/opt/spire/conf/agent/dummy_root_ca.crt"
+3. Set the value to  **/opt/spire/conf/agent/dummy_root_ca.crt**
 
 ### Configure Agent Plugins
 
 As described above in the [Plan Your Configuration](#plan-your-configuration] section, before installing and configuring you determine which plugin you configure the agent to use for node attestation and workload attestation. 
 
 #### Node Attestor Plugin
 
-IMPORTANT: The agent node attestor plugin must match the node attestor plugin type you choose when you configured the server.
+{{< warning >}}
```

Comment:
I should write up some docs on this, but I've added four "admonition" types to the template: info, warning, success, and danger.

Reply:
LGTM.